index.html

```diff
@@ -4266,9 +4266,10 @@ <h3>Securing Mechanism Specifications</h3>
 [=default graph=].
```

Comment:
I can't make a suggestion here, but I believe this should not be a statement about a "property" but about a securing mechanism, and it should be simplified to "The securing mechanism MUST secure all graphs in a [=verifiable credential=]." There's a minor technicality in that, a proof value itself (in a proof graph itself) cannot be "self-protected", but maybe that's obvious.

Similarly, the above "The property MUST define" seems more like it should say "The securing mechanism MUST define". This list should perhaps be about the requirements for the property *and* the new securing mechanism.

Reply:
I wonder whether —
```
Securing mechanism specifications that create new types of [=embedded proofs=] 
MUST specify a [=property=] for securing
```

— should become the following (which implies further changes to the bullet list, and possibly more) — 
```
Securing mechanism specifications that create new types of [=embedded proofs=]
MUST specify an object for securing both [=verifiable credentials=] and
[=verifiable presentations=].
```
— (should `object` be `[=object=]`?) with appropriate bullet adjustments, such as — 
```
The object MUST provide values for all [=properties] used by the [=proof graph=].
```

Reply:
I not against making this change, but that is more complex change than what this PR proposes to do. I would prefer to finalize this PR first and, if needed, do that in a separate PR.

Reply:
B.t.w., @TallTed, the (RDF) "object" term is not used in the core VCDM terminology, only "value".